source/intro.tex

```diff
@@ -14,7 +14,7 @@
 \Cpp{} is a general purpose programming language based on the C
 programming language as described in ISO/IEC 9899:2011
 \doccite{Programming languages --- C} (hereinafter referred to as the
-\defnx{C standard}{C!standard}). In addition to
+\defnx{C standard}{C!standard}). In addition to some of
```

Comment:
Hmm, how about:

> C++ provides many facilities beyond those provided by C, including [...]

This simultaneously minimizes the suggestion that C++ is a strict superset of C, and removes the suggestion that the following list of C++ features is a complete list of features in C++ but not C.

Reply:
Done.